packages/launcher/src/widget.tsx

```diff
@@ -298,7 +299,11 @@ function Card(
       <div className="jp-LauncherCard-icon">
         {kernel ? (
           item.kernelIconUrl ? (
-            <img src={item.kernelIconUrl} className="jp-Launcher-kernelIcon" />
+            <img
+              src={item.kernelIconUrl}
+              className="jp-Launcher-kernelIcon"
+              alt={trans.__('Create new notebook using kernel')}
```

Comment:
Let's use the title here as the card may be use for creating notebook or console.

```suggestion
              alt={title}
```